File: reactjs/src/app/groups/group5/services/dto/jobSeekerDto/experienceDto.ts
export interface CreateExperienceInput {
    company: string,
    startYear: number,
    endYear: number,
    role: string,
    description: string,
    idJobSeeker: number,
    grantedPermissions: [
        string
    ]
}

export interface UpdateExperienceInput extends CreateExperienceInput {
    id: number
}

export interface GetAllExperienceByIDOutput<IExperienceItem> {
    items: IExperienceItem[]
}

File: aspnet-core/src/group5/Group5.SE347.L11_HelloWork.Application/Services/Experiences/Dto/ExperienceDto.cs
using Abp.Application.Services.Dto;
using Abp.AutoMapper;
using SE347.L11_HelloWork.Entities;
using System;
using System.Collections.Generic;
using System.Text;

namespace Group5.SE347.L11_HelloWork.Application.Services.Experiences.Dto
{
    [AutoMapFrom(typeof(Experience))]
    public class ExperienceDto : EntityDto<int>
    {
        public string Company { get; set; }
        public int StartYear { get; set; }
        public int EndYear { get; set; }
        public string Role { get; set; }
        public string Description { get; set; }
        public long? IDJobSeeker { get; set; }
    }
}
